package.json

```diff
@@ -28,6 +29,7 @@
     "react-scripts": "3.4.1",
     "serve": "^11.3.1",
     "sharethis-reactjs": "^1.5.0",
+    "styled-components": "^5.1.1",
```

Comment:
Is this intentional to pull two different versions?

Reply:
removed.